projects/experimental/directives/button-group/button-group.style.less

```diff
@@ -0,0 +1,72 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroup] {
+    display: flex;
+    border-radius: 1.5rem;
+    justify-content: center;
+    overflow: hidden;
+
+    > button {
+        .button-base();
+
+        position: relative;
+        flex-grow: 1;
+        flex-basis: 0;
+        border-radius: 0;
+        flex-direction: column;
+        padding: 1.125rem 0.25rem 1.125rem 0.25rem;
+        gap: 0.5rem;
+        height: auto;
+        font-size: 0.8125rem;
+        white-space: pre-line;
+        min-width: 7.125rem;
+        font-weight: normal;
+        max-width: calc(50% - 1.75rem);
+        overflow: visible;
+        cursor: pointer;
+        color: var(--tui-link);
+
+        &:active,
+        &:active:before,
+        &:active:after {
+            background: var(--tui-clear);
+        }
+
+        &:before,
+        &:after {
+            position: absolute;
+            top: 0;
+            width: 1.75rem;
+            height: 100%;
+        }
+
+        &,
+        &:before,
+        &:after {
+            transition: background var(--tui-duration, 300ms) ease-in-out;
```

Comment:
Please use mixin. Also check if transition is really needed on pseudo elements after we add `background: inherit`.

Reply:
fixed